packages/camera/camera/example/integration_test/camera_test.dart

```diff
@@ -275,7 +275,9 @@ void main() {
     await controller.initialize();
     await controller.prepareForVideoRecording();
 
-    await controller.startVideoRecording();
+    await controller.startVideoRecording(
+      enableAndroidPersistentRecording: true,
```

Comment:
Assuming `enableAndroidPersistentRecording` is kept in this PR, this probably needs to be gated by a check for the Android platform.